core/src/main/java/org/lflang/LinguaFranca.xtext

```diff
@@ -218,6 +218,7 @@ Deadline:
     'deadline' '(' delay=Expression ')' code=Code;
 
 Watchdog:
+    (attributes+=Attribute)*
```

Comment:
I'm surprised to see a grammar change in a PR about diagram support. Why this change?

Reply:
So that you can add annotations, like labels and layout annotations.

Reply:
:+1: